app/src/main/java/org/mifos/mobilebanking/ui/fragments/BeneficiaryApplicationFragment.java

```diff
@@ -286,10 +287,12 @@ public void showBeneficiaryCreatedSuccessfully() {
      */
     @Override
     public void showBeneficiaryUpdatedSuccessfully() {
-        Toaster.show(rootView, getString(R.string.beneficiary_updated_successfully));
+        String className = BeneficiaryListFragment.class.getName();
+        Fragment fragment = getActivity().getSupportFragmentManager().findFragmentByTag(className);
+        BeneficiaryListFragment castedListFragment = (BeneficiaryListFragment) fragment;
+        castedListFragment.showSnackbar(getString(R.string.beneficiary_updated_successfully));
```

Comment:
Don't create variables, do this without any declaring variable in single line. 
For example `(new Object()).showSnackbar()`.